complex_spatial_process.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import os
</pre><p>either comment out or remove this cell since I believe we don't really need it? It's just for comparing with the weather data right? I think it's best to make it clear that this notebook is for dividing up the area</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>